src/AppBundle/Form/CommentType.php

```diff
@@ -54,9 +47,6 @@ public function configureOptions(OptionsResolver $resolver)
         ));
     }
 
-    /**
-     * @return string
-     */
     public function getName()
```

Comment:
However, if this method has only `{@inheritdoc}` - it also seems that method hasn't a return value. But `{@inheritdoc}` considers as a better way than duplicate annotation from a superclass.